server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -611,6 +611,26 @@ void sc_osc_handler::open_udp_socket(ip::address address, unsigned int port) {
     else
         sc_notify_observers::udp_socket.open(udp::v4());
 
```

Comment:
See my comments in the other file. The only exception: supernova uses snake case, so you can keep it here!